README.md

```diff
@@ -6,6 +6,23 @@ whatwg-url is a full implementation of the WHATWG [URL Standard](https://url.spe
 
 whatwg-url is currently up to date with the URL spec up to commit [6ef17eb](https://github.com/whatwg/url/commit/6ef17ebe1220a7e7c0cfff0785017502ee18808b).
 
+## Instructions
+
+First, install `npm`.
+
+    npm update -D
```

Comment:
Not quite. You'd want to `npm install` (`-D`/`--save-dev` is useful when you are installing new packages), and it's "install the dependencies of this package" rather than `npm` itself.

Reply:
Ah, so `npm install` fetches all the dependencies of the package into the local `node_modules` directory? I fail to see the difference between that and `npm update -D` but it seems to work, so changed.

> it's "install the dependencies of this package" rather than `npm` itself.

Sorry, I didn't mean that previous line to be a description of this command. I meant "first install npm, then run this command". Updated the text to make that clear.

Reply:
> I fail to see the difference between that and `npm update -D`

I don't think there is a practical difference but `npm install` is more inline with the intent of this action.